source/PreferencesPanel.cpp

```diff
@@ -178,6 +198,29 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		if(zones[latest].Value().KeyName() != Command::MENU.KeyName())
 			Command::SetKey(zones[latest].Value(), 0);
 	}
+	else if(key == 'i' && page == 'p')
+	{
+		page = 'i';
+		if(!downloadedInfo)
+		{
+			PluginHelper::Download(PLUGIN_LIST_URL.c_str(), (Files::Config() + "plugins.json").c_str());
+			ifstream pluginlistFile(Files::Config() + "plugins.json");
+			pluginInstallList = nlohmann::json::parse(pluginlistFile);
+			pluginlistFile.close();
```

Comment:
The close call is unnecessary, the file will get automated closed when the variable is deatroyed

Reply:
removed